recursion/circuit/src/stark.rs

```diff
@@ -361,16 +363,142 @@ pub fn build_wrap_circuit(
     backend.emit(builder.operations)
 }
 
+/// A function to build the Plonk constraints for a proof and a verifying key. The main difference
+/// between this function and the "old" one is that we use the RecursionAir machine from core-v2.
+pub fn build_wrap_circuit_new<F, const DEGREE: usize, const COL_PADDING: usize>(
```

Comment:
build_wrap_circuit_v2?

Reply:
also i feel like the comment can be made "more professional" / "concise".

/// A function to build the circuit for the wrap layer using core-v2.